quickwit/quickwit-indexing/src/actors/packager.rs

```diff
@@ -328,6 +331,47 @@ fn create_packaged_split(
     Ok(packaged_split)
 }
 
+/// Serializes the Split fields.
+///
+/// `fields_metadata` has to be sorted.
+pub fn serialize_field_metadata(fields_metadata: &[FieldMetadata]) -> Vec<u8> {
```

Comment:
does it need to be pub?

Reply:
no, that's an artifact from moving the code